graphql/fetchUserAssignmentSubmissions.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from "@apollo/client";
+export const FETCH_USER_ASSIGNMENT_SUBMISSIONS = gql`
+  query fetchUserAssignmentSubmissions($user_id: String = "") {
+    user_assignment_submissions(where: { user_id: { _eq: $user_id } }) {
+      submission_link
+      id
```

Comment:
I pulled data for all the fields in the table in graphQL. wasn't sure if that was good so that I could use the same query later if needed or if it would be better just to query and set types for the one field I needed for this task.

Reply:
see comment below about the response being an array